### PR TITLE
Add example to documentation about 'show_in_dashboard' option.

### PR DESCRIPTION
**Pull request by tim96**

### CODE REVIEW

File: Resources/doc/cookbook/recipe_knp_menu.rst [hunk 1]
```diff
@@ -149,6 +149,38 @@ You can modify the menu via events easily. You can register as many listeners as
 
 Please see the `Using events to allow a menu to be extended`_ for further information.
 
+
+Hide menu item
+------------------
+
+You can modify the menu to hide some menu item. You need to add option 'show_in_dashboard' in your admin services or simple remove menu item from sonata_admin dashboard group configuration:
```
Comment (greg0ire) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: Please line break this, and use double backticks around `sonata_admin`

File: Resources/doc/cookbook/recipe_knp_menu.rst [hunk 2]
```diff
@@ -149,6 +149,38 @@ You can modify the menu via events easily. You can register as many listeners as
 
 Please see the `Using events to allow a menu to be extended`_ for further information.
 
+
+Hide menu item
```
Comment (greg0ire) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: Hid**ing** menu item**s**

File: Resources/doc/cookbook/recipe_knp_menu.rst [hunk 3]
```diff
@@ -149,6 +149,38 @@ You can modify the menu via events easily. You can register as many listeners as
...
+------------------
+
+You can modify the menu to hide some menu item. You need to add option 'show_in_dashboard' in your admin services or simple remove menu item from sonata_admin dashboard group configuration:
+
+.. code-block:: php
```
Comment (greg0ire) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: This is not php.

File: Resources/doc/cookbook/recipe_knp_menu.rst [hunk 4]
```diff
@@ -149,6 +149,38 @@ You can modify the menu via events easily. You can register as many listeners as
...
+
+    tim_cheat_sheet.admin.log:
+        class: Sonata\AdminBundle\Admin\PostAdmin
+        arguments: [~, Sonata\AdminBundle\Entity\Post, SonataAdminBundle:CRUD]
+        tags:
+            - {name: sonata.admin, manager_type: orm, group: admin, label: Post, show_in_dashboard: false}
+
+.. code-block:: php
```
Comment (greg0ire) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: Neither is this

File: Resources/doc/cookbook/recipe_knp_menu.rst [hunk 5]
```diff
@@ -149,6 +149,38 @@ You can modify the menu via events easily. You can register as many listeners as
...
+    # app/config/config.yml
+
+    sonata_admin:
+        dashboard:
+            groups:
+                news:
+                    label:                ~
+                    label_catalogue:      ~
+                    items:
+                      # just comment or remove declaration about `sonata.news.admin.post` to hide it from menu
```
Comment (greg0ire) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: > just comment or remove the `sonata.news.admin.post` declaration to hide it from the menu.

**Comment by tim96** [comment]
@greg0ire thank you. Fixed.

File: Resources/doc/cookbook/recipe_knp_menu.rst (hunk 1 above)
Comment (tim96) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: > Please linebreak this

@greg0ire How I can do it?

File: Resources/doc/cookbook/recipe_knp_menu.rst [hunk 6]
```diff
@@ -149,6 +149,37 @@ You can modify the menu via events easily. You can register as many listeners as
...
 Please see the `Using events to allow a menu to be extended`_ for further information.
 
+Hiding menu items
+------------------
+
+You can modify the menu to hide some menu items. You need to add the ``show_in_dashboard`` option in your admin services or simply remove menu items from the ``sonata_admin`` dashboard group configuration:
+
+.. code-block:: yaml
+
+    tim_cheat_sheet.admin.log:
```
Comment (core23) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: Could you use a general dummy name like ``acme.admin.log``?

**Comment by core23** [comment]
Refs https://github.com/sonata-project/SonataAdminBundle/pull/3483

File: Resources/doc/cookbook/recipe_knp_menu.rst (hunk 6 above)
Comment (tim96) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: @core23 sure, sorry I take this example from other project.

File: Resources/doc/cookbook/recipe_knp_menu.rst (hunk 1 above)
Comment (greg0ire) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: Like this : http://stackoverflow.com/questions/1272173/in-vim-how-do-i-break-one-really-long-line-into-multiple-lines

Comment (tim96) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: @greg0ire I mean why I need to do it? I don't find any lines with linebreak in this document. This document contains a line much more longer. Is it some new rule about line length?

Comment (greg0ire) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: > I don't find any lines with linebreak in this document. This document contains a line much more longer.

I know, they are all over the place :(

[I try to fix that](https://github.com/sonata-project/SonataAdminBundle/commit/d06f9568fca973e707f69a56687b46ebc9ae0590), but it is a huge job, so let's just not add new ones if we can, ok ?

> Is it some new rule about line length?

Yes. Try to keep them under 120 chars if you can, or even under 80 if possible.

Comment (tim96) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: > I try to fix that, but it is a huge job, so let's just not add new ones if we can, ok ?

@greg0ire sure. Fixed it.

Comment (greg0ire) [inline Resources/doc/cookbook/recipe_knp_menu.rst]: Thanks :)

**Comment by greg0ire** [comment]
LGTM . Either squash your commits into one or let the merger do it for you.

**Comment by tim96** [comment]
> Either squash your commits into one

Fixed

**Comment by greg0ire** [comment]
Thanks @tim86

**Pull request merged**